### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by frederikhors** [sqlc-dev/sqlc#581] [Issue Triage] [answered]
What do you think of using [docusaurus.io](https://docusaurus.io) for the documentation on the site?

We can help you create it through PR.

It would be great to have a "recipes" section for learning how to write sql queries for example.

**Comment by kyleconroy** [comment, marked as answer]
I haven't made a decision about what documentation system to use. I setup a GitBook, but didn't really like that fact that it was paid and I couldn't edit things via code. I'll take a look at docusaurus.io and see if it meets my needs.
